src/main/java/com/project/fruits_ecommerce/entities/User.java

```diff
@@ -0,0 +1,23 @@
+package com.project.fruits_ecommerce.entities;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Data
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String username;
+    private String password;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
Let's make the fetchType LAZY instead for performance, and load them when needed